### DISCUSSION TITLE
Your license is not allowed to download this release!

### DISCUSSION THREAD

**Discussion by exshapoval** [laravel/nova-issues#5490] [Q&A] [answered]
- Laravel Version: 10.6.2
- Nova Version: 4.22.2
- PHP Version: 8.2.3

### Description:
try to run composer update
and after this see error in cycle

Your license is not allowed to download this release!

How to fix it?

**Comment by crynobone** [comment, marked as answer]
You need to lock the dependency to last supported version before the updates expired, in this case it should be `4.22.2`:

```
composer require "laravel/nova:4.22.2"
```
